pom.xml

```
@@ -341,6 +352,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
+                <version>${maven-javadoc-plugin.version}</version>
                 <configuration>
                     <additionalOptions>
                         <additionalOption>-Xdoclint:none</additionalOption>
```

I see that this was updated on a recent commit (it had `<additionalparam>` before).

If I had to guess, I'd say that change fixed javadoc generation locally, but broke it in on sonatype,
and this commit updates the maven-javadoc-plugin so that it will work again?

Anyway, if we're using the plugin v3.0.0, there is a specific tag for this.
Instead of passing options to the javadoc tool explicitly, we can simply use the _doclint_  tag
```suggestion
                    <doclint>none</doclint>
```
(I couldn't include the `</additionalOptions>` line in the suggestion, but if you accept it, then that line has to be removed too)

---

> If I had to guess, I'd say that change fixed javadoc generation locally, but broke it in on sonatype,
and this commit updates the maven-javadoc-plugin so that it will work again?
That's correct.

> Anyway, if we're using the plugin v3.0.0, there is a specific tag for this.
> Instead of passing options to the javadoc tool explicitly, we can simply use the doclint tag
Nice! Changed accordingly.